cg/cli/upload/base.py

```diff
@@ -39,23 +38,22 @@
 from cg.meta.upload.upload_api import UploadAPI
 from cg.models.cg_config import CGConfig
 from cg.store import Store
-
 from cg.store.models import Family
 from cg.utils.click.EnumChoice import EnumChoice
 
 LOG = logging.getLogger(__name__)
 
 
 @click.group(invoke_without_command=True)
-@click.option("-f", "--family", "family_id", help="Upload to all apps")
+@click.option("-c", "--case", "case_id", help="Upload to all apps")
 @click.option(
     "-r",
     "--restart",
     is_flag=True,
     help="Force upload of an analysis that has already been uploaded or marked as started",
 )
 @click.pass_context
-def upload(context: click.Context, family_id: Optional[str], restart: bool):
+def upload(context: click.Context, case_id: Optional[str], restart: bool):
```

Comment:
```case_id``` is in this case ```case_internal_id``` which refers to the case name e.g. "updog" 

there was a discussion some time ago how we wanted to refer to such ids and the outcome was to be more explicit at least in the backend. I am not sure what we want in the cli with which the users interact. 

@henrikstranneheim, @Vince-janv, @seallard and @diitaz93 what are your thoughts?

Reply:
Since the field on the model is called `internal_id` I think we should refer to the variable as `case_internal_id`. I don't really think `internal_id` is a good name, but that is a separate issue.

However, I don't think the user of the CLI should know about these hair cleaving exercises and for them it would make much more sense to just refer to it as the case id.

Reply:
I think we should make as few changes as possible to the CLI. I would say keep `case_id` and leave the `internal_id`s inside infrastructure

Reply:
@diitaz93 The CLI used family_id before though, but maybe you meant that starting with case_internal_id would imply changes to the CLI in the future to match all other usages of case_id?

Reply:
```suggestion
def upload(context: click.Context, case_internal_id: Optional[str], restart: bool)
```
so we settle for this then? since the parameter has to be changed from ```familiy_id``` to something with ```case```anyway we can be explicit and make it ```case_internal_id```?

Reply:
yes! Sorry for not being clear. The change of family to case has to be done. But changing "id" to "internal_id" would bring unnecessary confusion. I would prefer just `_id` just in this case as it is far away from the SQLAlchemy models, but I would understand why @ChrOertlin wants to keep the ìnternal`, it is "more correct" but less practical I think.

Reply:
To me, it would be more confusing to have case_id in most places and case_internal_id in just one, although I agree that case_internal_id is probably better in the long-run. So my suggestion is that we use case_id for now and then open a separate pr where we do a larger overhaul of the naming. @ChrOertlin @diitaz93.